### ISSUE TITLE
[a11y] Cannot create record set from a query

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1489]
![image](https://user-images.githubusercontent.com/37256050/167661055-41b163d8-b940-44d3-960a-4d27d4ce766b.png)

The **Save** option is disabled. In `edge` this feature is working correctly! Same query (AllAgentQuery).

db: fwri
user: plarson
col: ichtyhology

**Comment by acbentley** [comment]
This is not just for agents but for any query.

**Comment by maxxxxxdlp** [comment]
Duplicate of #1480
